remme/clients/pub_key.py

```diff
@@ -24,6 +24,7 @@
 from remme.tp.account import AccountHandler
 from remme.tp.pub_key import PUB_KEY_ORGANIZATION, PUB_KEY_MAX_VALIDITY
 from remme.settings.helper import _make_settings_key
+from remme.settings import GENESIS_ADDRESS
```

Comment:
It is not good that this address is hardcoded. Better move it to settings & set on genesis block creation.